source/connect/favorite-connections/import-export-ui/import.txt

```diff
@@ -0,0 +1,57 @@
+.. _compass-import-connections-ui:
+
+==============================================
+Import Favorite Connections in |compass-short|
+==============================================
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+You can import favorite connections in |compass|. This lets you use saved 
+connections from other workspaces or connections used by other team members.
+
+Prerequisite
+------------
+
+Before you import favorite connections, you must :ref:`export the
+connections <compass-export-connections-ui>`.
+
+Procedure
+~~~~~~~~~
```

Comment:
Same issue here with these headings all appearing as sub-headings of "Prerequisite". We'll want to use dash underlines throughout.

Reply:
Updated.